spec/requests/employee_invitation_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+RSpec.describe 'EmployeeInvitationController', type: :request do
+  let(:user) { create(:user, :admin) }
+  let(:not_listed_user) { create(:user) }
+  let(:company) { create(:company) }
+  let(:headers) { { Authorization: sign_in(user) } }
+
+  describe 'POST /companies/:company_id/employee_invitations' do
+    subject(:invitation_controller) do
+      post company_employee_invitation_path(company),
+        params: { email: current_email }, headers: headers
+    end
+
+    context 'when the user is not registered on comarev' do
+      let(:current_email) { 'test_test@email.com' }
+
+      before do
+        allow(CheckEmployeeAssociation).to receive(:call)
+          .and_return({ message: 'Email sent to new user', status: :ok })
+      end
+
+      it "returns status ok with 'email sent to new user' message" do
+        invitation_controller
+
+        expect(response).to have_http_status(:ok)
+        expect(response.body).to eq({ message: 'Email sent to new user' }
+          .to_json)
+      end
+    end
+
+    context 'when the user is already listed on a company' do
+      let(:current_email) { user.email }
+
+      before do
+        create(:company_user, user: user, company: company)
+        allow(CheckEmployeeAssociation).to receive(:call)
+          .and_return({ message: "User is already listed as #{company.name}'s employee",
+            status: :unprocessable_entity })
+      end
+
+      it "returns status unprocessable entity with 'already listed' message" do
+        invitation_controller
+
+        expect(response).to have_http_status(:unprocessable_entity)
+        expect(response.body).to eq({ message:
+          "User is already listed as #{company.name}'s employee" }.to_json)
```

Comment:
Maybe you will agree with me, it seems a liitle weird this key `message` alone 
```suggestion
        expect(response.body).to eq(
          { message: "User is already listed as #{company.name}'s employee" }.to_json
        )
```